### PR TITLE
docs: polish wording in NatSpec

### PR DESCRIPTION
**Pull request by PaulRBerg**

### CODE REVIEW

File: src/core/types/DataTypes.sol [hunk 1]
```diff
@@ -103,18 +103,18 @@ library Lockup {
 
 /// @notice Namespace for the structs used in {SablierLockupDynamic}.
 library LockupDynamic {
-    /// @notice Struct encapsulating the parameters of the {SablierLockupDynamic.createWithDurations} function.
-    /// @param sender The address distributing the assets, with the ability to cancel the stream. It doesn't have to be
-    /// the same as `msg.sender`.
-    /// @param recipient The address receiving the assets.
-    /// @param totalAmount The total amount of ERC-20 assets to be distributed, including the stream deposit and any
-    /// broker fee, both denoted in units of the asset's decimals.
+    /// @notice Struct encapsulating the parameters of the {SablierV2LockupDynamic.createWithDurations} function.
```
Comment (smol-ninja) [inline src/core/types/DataTypes.sol:106]: should be `SablierLockupDynamic`.

File: src/core/types/DataTypes.sol [hunk 2]
```diff
@@ -127,18 +127,18 @@ library LockupDynamic {
         Broker broker;
     }
 
-    /// @notice Struct encapsulating the parameters of the {SablierLockupDynamic.createWithTimestamps} function.
-    /// @param sender The address distributing the assets, with the ability to cancel the stream. It doesn't have to be
-    /// the same as `msg.sender`.
-    /// @param recipient The address receiving the assets.
-    /// @param totalAmount The total amount of ERC-20 assets to be distributed, including the stream deposit and any
-    /// broker fee, both denoted in units of the asset's decimals.
+    /// @notice Struct encapsulating the parameters of the {SablierV2LockupDynamic.createWithTimestamps} function.
```
Comment (smol-ninja) [inline src/core/types/DataTypes.sol:130]: should be `SablierLockupDynamic`.

File: src/core/types/DataTypes.sol [hunk 3]
```diff
@@ -163,8 +163,8 @@ library LockupDynamic {
         uint40 timestamp;
     }
 
-    /// @notice Segment struct used at runtime in {SablierLockupDynamic.createWithDurations}.
-    /// @param amount The amount of assets to be streamed in the segment, denoted in units of the asset's decimals.
+    /// @notice Segment struct used at runtime in {SablierV2LockupDynamic.createWithDurations}.
```
Comment (smol-ninja) [inline src/core/types/DataTypes.sol:166]: should be `SablierLockupDynamic`.

**Review by smol-ninja — CHANGES_REQUESTED** [review]
I like the polished version.

**Comment by PaulRBerg** [comment]
Thanks for the review. Fixed the issues.

**Comment by andreivladbrg** [comment]
Will merge it first, since it is a small PR and it causes no conflicts in the claim dry fy.

**Pull request merged**